Source/WebCore/fileapi/File.cpp

```diff
@@ -45,7 +45,7 @@ Ref<File> File::createWithRelativePath(ScriptExecutionContext* context, const St
     return file;
 }
 
-Ref<File> File::create(ScriptExecutionContext* context, const String& path, const String& replacementPath, const String& nameOverride)
+Ref<File> File::create(ScriptExecutionContext* context, const String& path, const String& replacementPath, const String& nameOverride, const std::optional<int64_t>& lastModified)
```

Comment:
Why is this a const reference instead of just a optional<int64_t>?

Reply:
because in the `deserialize` it's a const reference so I figured I would do the same; I can change it to not be a const ref if you want?

Reply:
Which was from Chris in https://trac.webkit.org/changeset/232043/webkit. No, you can leave it.